### ISSUE TITLE
Consider capturing which config format is used

### ISSUE THREAD

**Issue by cartermp** [honeycombio/refinery#518]
We get feedback from our friends in the field that yaml configs are used a ton, especially when using refinery with k8s. But we can quantify how many use which config! Maybe that's worth doing.

**Comment by kentquirk** [comment]
Because viper transparently loads all sorts of file types and doesn't tell us what it did, about all we could do is log the filename specified in the configuration.

But I guess the only way it knows the file type to parse is by the file extension, so if we record the extension it's probably close enough.

**Comment by cartermp** [comment]
Yeah, it's probably a safe assumption that if the file name ends with `.yaml` or `.toml` it's one or the other.
